explorer/templates_test.go

```diff
@@ -0,0 +1,59 @@
+package explorer
+
+import (
+	"testing"
+)
+
+func TestamountAsDecimalPartsTrimmed(t *testing.T) {
```

Comment:
This test will never run.  The first letter after "Test" must not be lowercase.  Rename to `TestAmountAsDecimalPartsTrimmed`